lib/datadog/core/remote/client.rb

```diff
@@ -4,14 +4,45 @@
 
 require_relative 'configuration'
 require_relative 'dispatcher'
-require_relative '../../appsec/processor/rule_merger'
-require_relative '../../appsec/processor/rule_loader'
 
 module Datadog
   module Core
     module Remote
       # Client communicates with the agent and sync remote configuration
       class Client
+        class SyncError < StandardError; end
+
+        class << self
+          attr_reader :products, :capabilities, :receivers
+
+          def init
+            @capabilities = []
+            @products = []
+            @receivers = []
+          end
```

Comment:
This looks like it should be an actual class, and have an instance of it in the client.